### PR TITLE
patches

### PR DESCRIPTION
**Pull request by procodeing**
This pull request fixes some issues with non-width-defined elements, and shortens the code. More patches to come, with explanations at the pen.
The pen is: http://codepen.io/procodeing/pen/BNagZa/

*Also maybe some 3d animations coming soon...*

### CODE REVIEW

**Comment by daneden** [comment]
@procodeing Hi! Could you give a description of the changes in this PR?

Also, I see you updated animate.css—you should change each of the files in /source to ensure the changes stick when animate.css is recompiled.

**Comment by procodeing** [comment]
@daneden sure, and i'll also upload a pen of what this does

**Comment by procodeing** [comment]
@daneden It should now be completly mergible.

**Comment by daneden** [comment]
@procodeing you should remove the prefixed properties from the source files—autoprefixer takes care of this stuff in the Grunt file :)

**Comment by procodeing** [comment]
@daneden sure.

**Comment by procodeing** [comment]
@daneden done

**Comment by procodeing** [comment]
@daneden its not the best fix, sencie it just make it pulse the other way

**Comment by procodeing** [comment]
Wait, I think i found a better fix

**Comment by c9n** [comment]
Why to change `scale3d` to `scale`?

**Comment by procodeing** [comment]
because the Z property wasn't being used, and it remained 1. scale3d(x,y,1) is the same thing as scale(x,y). Also, scale3d isn't supported in earlier versions of IE. Also, in my new patch, I use animated centerAnimation to make the animated element an inline block.

**Comment by daneden** [comment]
@procodeing that's not entirely true: `scale3d` forces the element to be animated/transformed using the GPU. That said, I'd not adverse to these changes on that basis: people should be adopting the new `will-change` property to enable hardware-accelerated transformations.

**Comment by procodeing** [comment]
oh

**Comment by procodeing** [comment]
__***the more you know***__

File: source/_base.css [hunk 1]
```diff
@@ -11,6 +11,10 @@ Copyright (c) 2015 Daniel Eden
   animation-fill-mode: both;
 }
 
+.animated.centerAnimation{
```
Comment (daneden) [inline source/_base.css]: This doesn't appear to be present in the compiled animate.css, and it's also relying on a lot of other CSS properties to do what (I think) it's intended to do. Remove this whole declaration.

File: source/attention_seekers/pulse.css [hunk 2]
```diff
@@ -1,19 +1,25 @@
 /* originally authored by Nick Pettit - https://github.com/nickpettit/glide */
-
+/* rewrote by Procodeing - https://github.com/procodeing */
```
Comment (daneden) [inline source/attention_seekers/pulse.css]: This is hardly a rewrite; credit where credit's due, but the changes here are minor. You'll still be included in the contributors page on the repo if/when this is merged!

**Comment by daneden** [comment]
Added some inline comments. Make those changes & then I’ll merge!

File: README.md [hunk 3]
```diff
@@ -12,7 +12,7 @@
   </head>
   ```
 2. Add the class `animated` to the element you want to animate.
- You may also want to include the class `infinite` for an infinite loop.
+ You may also want to include the class `infinite` for an infinite loop, and if the element has no width, add the `centerAnimation` class to it.
```
Comment (daneden) [inline README.md]: Again, the CSS for this class depends on a lot of other styles for it to work as expected, so remove this from the readme.

**Comment by procodeing** [comment]
@daneden Ok!

**Comment by procodeing** [comment]
@daneden Since the changes are so small, I will post the fix to it up later, like on my (upcoming) blog.
Though I have been working on some nice animations myself, but I will get to that later

**Comment by daneden** [comment]
@procodeing sounds good. Thanks for working with my requests on this :)

**Comment by procodeing** [comment]
@daneden your'e welcome

**Pull request closed without merging**